medaka_bpm.py

```diff
@@ -24,14 +24,23 @@
 import src.setup as setup
 import src.segment_heart as segment_heart
 
+curr_dir = os.getcwd()
+
 import configparser
-curr_dir = os.path.dirname(os.path.abspath(__file__))
 config_path = os.path.join(curr_dir, 'config.ini')
-
 config = configparser.ConfigParser()
 config.read(config_path)
 
+# QC Analysis modules.
+sys.path.append(os.path.join(curr_dir, "qc_analysis"))
+from decision_tree.src import analysis
```

Comment:
Just a detail - I think it'd make sense to add the decision tree file in a separate folder ./data/ aside the ./scr/ and ./qc-analysis/ folder.

In a way it's not a source file, but a binary that's loaded and we'd keep the various qc-routines separate from the main program flow.